transforms/code/license_check/README.md

```diff
@@ -0,0 +1,11 @@
+# License Check 
+
+The License Check transform checks if the `license` of input data is in approved/denied list. It is implemented as per the set of [transform project conventions](../../README.md#transform-project-conventions) the following runtimes are available:
```

Comment:
Can you be more specific and say it selects only the specified licenses and filters out the other licenses..